### PR TITLE
Hubert / installing and configurating eslint package

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "eslint": "npx eslint *.js && npx eslint *.jsx && npx eslint *.tsx"
```
Comment (balakrishna-binary) [inline package.json:9]: I think we can set eslint to pick up all the extensions in one go

`npx eslint --ext .js,.jsx,.tsx`

Comment (balakrishna-binary) [inline package.json:9]: I think we can set eslint to pick up all the extensions in one go

`npx eslint --ext .js,.jsx,.tsx`

File: Homepage.jsx [hunk 2]
```diff
@@ -130,15 +130,15 @@ export default function HomePage() {
...
+                    <p>Manage users&apos; accounts</p>
                   </li>
                   <li>
                     <div className={styles.checklistIcon} />
                     <p>Monitor existing contracts</p>
                   </li>
                   <li>
                     <div className={styles.checklistIcon} />
-                    <p>View users' historical transactions</p>
+                    <p>View users&apos; historical transactions</p>
```
Comment (balakrishna-binary) [inline Homepage.jsx:141]: I think we should still be using a single quote(') which will improve readability. What's preventing us from using it?

File: AppRegistration.jsx [hunk 3]
```diff
@@ -59,7 +58,7 @@ export default function AppRegistration() {
                 </div></fieldset>
               <div className="expand-form">
                 <input id="expand_form" className="expand-form-checkbox" type="checkbox" autoComplete="off" />
-                <label htmlFor="admin-scope">I'd like to use OAuth or monetise my app</label>
+                <label htmlFor="admin-scope">I&apos;d like to use OAuth or monetise my app</label>
```
Comment (matin-binary) [inline AppRegistration.jsx:61]: We still need to have the single quote here.

File: AppRegistration.jsx [hunk 4]
```diff
@@ -115,8 +114,8 @@ export default function AppRegistration() {
                       </div>
                     </div>
                     <p>Bear in mind that you generally need only
-                      <b>'Trade'</b> and
-                      <b>'Trading information'</b> access.
+                      <b>&apos;Trade&apos;</b> and
+                      <b>&apos;Trading information&apos;</b> access.
```
Comment (matin-binary) [inline AppRegistration.jsx:118]: ditto

**Review by balakrishna-binary — APPROVED** [review]

**Review by matin-binary — APPROVED** [review]

**Pull request merged**